tests/auth_tests/test_tokens.py

```diff
@@ -32,6 +32,23 @@ def test_10265(self):
         tk2 = p0.make_token(reload)
         self.assertEqual(tk1, tk2)
 
+    def test_26615(self):
```

Comment:
Agree, was just following the pattern of the other tests, so those ([e.g.](https://github.com/django/django/commit/3d0dcd7f5af378d3ab6adb303b913e6c7b2e0ee5#L22)) should probably be edited as well.